libraries/AP_Winch/AP_Winch_Servo.cpp

```diff
@@ -1,13 +1,23 @@
 #include <AP_Winch/AP_Winch_Servo.h>
+#include <GCS_MAVLink/GCS.h>
 
 extern const AP_HAL::HAL& hal;
 
-void AP_Winch_Servo::init(const AP_WheelEncoder* wheel_encoder)
+// true if winch is healthy
+bool AP_Winch_Servo::healthy() const
 {
-    _wheel_encoder = wheel_encoder;
+    // return immediately if no servo is assigned to control the winch
+    if (!SRV_Channels::function_assigned(SRV_Channel::k_winch)) {
+        return false;
+    }
 
-    // set servo output range
-    SRV_Channels::set_angle(SRV_Channel::k_winch,  1000);
+    return true;
+}
+
+void AP_Winch_Servo::init()
+{
+    // initialise rc input and output
+    init_input_and_output();
```

Comment:
Should call superclass `init()` method instead.

Reply:
txs.  Fixed to call the superclass init

Reply:
Ah, we can just remove AP_Winch_Servo::init()..